techniques/general/G211.html

```diff
@@ -207,19 +204,19 @@ <h3>Example 6: Stacked Labels</h3>
 			&lt;input class="input" id="example-2" name="example-2" type="text" aria-describedby="example-2-hint"&gt;<br>
 		&lt;/form&gt;<br>
 	</code>
-				<p class="working-example"><a href="../../working-examples/label-in-name-general/example2.html">Working example of stacked labels</a></p>
+				<p class="working-example"><a href="https://www.w3.org/WAI/WCAG22/working-examples/label-in-name-general/example2.html">積み重ねられたラベルの動作する例</a></p>
```

Comment:
working exapmple の訳を、他の箇所と合わせて「動作例」にしました。
```suggestion
				<p class="working-example"><a href="https://www.w3.org/WAI/WCAG22/working-examples/label-in-name-general/example2.html">積み重ねられたラベルの動作例</a></p>
```